appveyor.yml

```diff
@@ -1,14 +1,14 @@
 environment:
-  nuget_version: '0.11.1'
+  nuget_version: '0.11.2'

```

Comment:
Thinking this should become 0.12 based on SemVer given that this is not a bug fix, but a backwards compatible feature.

Reply:
Noted

Reply:
I've updated this version to 0.12